image/templates/helm/stackrox-secured-cluster/internal/config-shape.yaml

```diff
@@ -144,7 +144,9 @@ customize:
   other: {} # dict
 allowNonstandardNamespace: null # bool
 allowNonstandardReleaseName: null # bool
-enableOpenShiftMonitoring: null # bool
```

Comment:
This looks like a breaking change. A customer having `enableOpenShiftMonitoring=true` would after an upgrade set the value to `false`.
Can you add logic to the `_init.tpl.htpl` to add a backward compatibility check which translates the old value to the new one?

Reply:
Thoughts about deprecating and removing `enableOpenShiftMonitoring`? As far as I understand it was never documented or intended for customer use - it was just added for internal testing. So deprecation should easy. The reason we renamed it is purely because we prefer the `category.enabled` style of flags.

Reply:
**Compatibility**
Backwards compatibility is a must and easy to implement here. 

I see two options:
 - Stay with `enableOpenShiftMonitoring`
 - Add backward compatibility

If the field exists there is always the possibility that a customer or solution architect used it as a fix. We can mitigate this breaking change by translating the config.
Adding a deprecation notice to remove it with the next major release sounds good.

Reply:
@janisz already implemented the backwards translation check 👍